proto/vendor/protobuf/src/google/protobuf/empty.proto

```diff
@@ -40,7 +40,7 @@ option objc_class_prefix = "GPB";
 option csharp_namespace = "Google.Protobuf.WellKnownTypes";
 option cc_enable_arenas = true;
 
-// A generic empty message that you can re-use to avoid defining duplicated
+// A generic empty message that you can reuse to avoid defining duplicated
```

Comment:
This file is not what we controlled, so no need to fix the codespell issue here. I updated codespell config to skip proto folder.

Reply:
Fixed, thanks!